src/api/Dockerfile

```diff
@@ -1,30 +1,75 @@
-# Node Alpine Docker Image
-FROM node:18.9-alpine3.16
+###################
+# 1. BUILD
+###################
 
-# Variables
+# NPM 8.19.1 Alpine linux image
+FROM node:18.15-alpine3.16 AS build
```

Comment:
keep an eye in Azure to ensure that this version of Node is supported

Reply:
Will do.